bin/update-gitignore.php

```diff
@@ -0,0 +1,12 @@
+<?php
+/**
+ * @link      https://github.com/zendframework/ZendSkeletonApplication for the canonical source repository
+ * @copyright Copyright (c) 2019 Zend Technologies USA Inc. (https://www.zend.com)
+ * @license   https://framework.zend.com/license/new-bsd New BSD License
```

Comment:
we are using link to LICENSE.md of the repository - as the above does not work

Reply:
Updated.